aplib.net-demo/Assets/Materials/Colors/Dark Blue Mat.mat

```diff
@@ -127,7 +127,7 @@ Material:
     - _ZWrite: 1
     m_Colors:
     - _BaseColor: {r: 0.068629384, g: 0.45377085, b: 0.7132075, a: 1}
-    - _Color: {r: 0.06862936, g: 0.45377082, b: 0.7132074, a: 1}
+    - _Color: {r: 0.06862936, g: 0.45377082, b: 0.7132075, a: 1}
```

Comment:
no

Reply:
There is no escaping this world of portability